tests/grid_test.cc

```diff
@@ -96,17 +97,10 @@ void testGrid( const GridView& gridView )
         std::cout << "number of elements is wrong: " << numElem << "\n";
 }
 
-int main()
+template <class Grid>
+void testGrid(Grid& grid, const std::string& name)
 {
-    Opm::Parser parser;
-    Opm::ParseMode parseMode;
-    const auto deck = parser.parseString(deckString , parseMode);
-
-    std::vector<double> porv;
-    typedef Dune::PolyhedralGrid< 3, 3 > Grid;
-    typedef Grid::LeafGridView GridView;
-    Grid grid(deck, porv);
-
+    typedef typename Grid::LeafGridView GridView;
 #if DUNE_VERSION_NEWER(DUNE_GRID,3,0)
```

Comment:
I need to ask (again, sorry): is DUNE_VERSION_NEWER really NEWER_OR_EQUAL_TO? That is, this will trigger for version 3.0? Also, I assume that 3.0 is the current git master version, so this will not trigger for the 2.4 release. Is that intended, or is this something remaining from a time when we thought there would be no 2.4 release?